scanner/cmd/updater/main.go

```diff
@@ -16,7 +16,7 @@ import (
 const DefaultURL = "https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml"
 
 func tryExport(ctx context.Context, outputDir string, opts *updater.ExportOptions) error {
-	const timeout = 2 * time.Hour
+	const timeout = 7 * time.Hour
```

Comment:
7 hours feels way too long. Why not 3?

Reply:
:+1:

Reply:
Also, can you please evaluate if setting this from an environment variable is straightforward and adopt that if it is? It allow adjusting without having to change the updater.

Reply:
It's taking way too long:
https://github.com/stackrox/stackrox/actions/runs/15350957753/job/43198829666